docs/book/features/vsphere_csi_migration.md

```diff
@@ -8,7 +8,7 @@
 
 ## Introduction <a id="introduction"></a>
 
-**Note:** vSphere CSI Driver images for this feature is not yet released. When images will be available for public consumption we will update this document.
+**Note:** Feature to migrate in-tree vSphere volumes to CSI migration is released as **beta** with [v2.0.1](https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.1.0).
```

Comment:
v2.0.1 ==> v2.1.0 ?

Reply:
I think we should rephrase:
`Feature to migrate in-tree vSphere volumes to CSI migration is`
to
`Feature to migrate in-tree vSphere volumes to CSI is`

Reply:
changing to

> **Note:** Feature to migrate in-tree vSphere volumes to CSI is released as **beta** with [v2.1.0](https://github.com/kubernetes-sigs/vsphere-csi-driver/releases/tag/v2.1.0).